content/docs/reference/config/installation-options/index_version_diff.md

```diff
@@ -0,0 +1,48 @@
+## `ingress` options
```

Comment:
You need a bit of front-matter here to supply title, description, etc. See other .md files in the repo.

Reply:
@geeknoid I added the meta to the md file. Thanks.